securedrop/source_app/__init__.py

```diff
@@ -79,8 +79,8 @@ def setup_g():
 
         if 'expires' in session and datetime.utcnow() >= session['expires']:
             session.clear()
-            flash(gettext('You have been logged out due to inactivity'),
-                  'error')
+            msg = render_template('session_timeout.html')
```

Comment:
Won't this change mean that users who were not using TB and were logged out will see nothing?